client/src/main/java/com/distkv/client/DistkvDictProxy.java

```diff
@@ -88,4 +88,16 @@ public void removeItem(String key, String itemKey) {
     DistkvResponse response = FutureUtils.get(asyncDictProxy.removeItem(key, itemKey));
     CheckStatusUtil.checkStatus(response.getStatus(), key, typeCode);
   }
+
+  /**
+   * Expire a key
+   *
+   * @param key The key to be expired
+   * @param expireTime Millisecond level to set expire
+   */
+  public void expire(String key, long expireTime) {
+    DistkvResponse response = FutureUtils.get(asyncDictProxy.expire(key, expireTime));
+    CheckStatusUtil.checkStatus(response.getStatus(), key, typeCode);
+  }
```

Comment:
Here is duplicate with DistkvIntProxy.java , DistkvListProxy.java ,DistkvSetProxy.java ,DistkvSortedListProxy.java ,DistkvStringProxy.java  too.

Reply:
Same as above